docs/source/contributing/developing.adoc

```diff
@@ -192,12 +192,19 @@ $go test -v <relative path of package> -run "Test<Regex pattern to match tests>"
 For more information about test options, run the `go test --help` command and review the documentation.
 
 [[integration-tests]]
-=== Running Integration Tests
+=== Integration Tests
 
-Integration tests utilize link:https://github.com/DATA-DOG/godog[Godog], which uses Gherkin (Cucumber) to define test cases.
-The test cases are defined in *_test/integration/features_*.
-By default, the tests are executed against the binary created by `make build`, that is *_$GOPATH/bin/minishift_*.
-To run the tests, run:
+Integration tests utilize link:https://github.com/DATA-DOG/godog[Godog], which uses Gherkin (Cucumber) to define sets of test cases,
+aka features in Gherkin terminology. The features are defined in *_test/integration/features_*. Features for Minishift are following
```

Comment:
> I think the path for features should be _test/integration/features/*.feature

Maybe, maybe not. Does not really matter so much imo. We are aiming though for a sentence a line. So, there is a "wrong" line break after the command and a missing one after _ terminology_.